README.md

```diff
@@ -1015,6 +911,97 @@ The `IfUser()` helper syntax is as follows:
 IfUser(attributeName: string, attributeValue: string | number | boolean);
 ```
 
+### Using the `UserManagement` addon
+
+In order to put all the pieces together, JSONAPI-TS provides an [addon](#what-is-an-addon) to manage both user and session concerns.
+
+You'll need to define at least two functions:
```

Comment:
I would make it clearer when both of these functions are used.
Something like:
"An `encryptPassword` callback that get's used in user creation process to set the password/security field"
and 
"A `login` callback that get's used in the login process"

I don't think that it's clear enough as it is to a just-arrived dev to understand

Reply:
Will improve per your suggestions :)